### PR TITLE
Initial work on main dishes details activity

### PR DESCRIPTION
**Pull request by WadeQ**

### CODE REVIEW

**Comment by WadeQ** [comment]
> Hello @WadeQ
> 
> Before I merge this in, kindly ensure that it merges into `deploy` branch and not `master` branch as it currently does. Thanks.

Can i have help with that? I think i have exhausted my knowledge here and i followed all steps you laid down.

**Comment by TheDancerCodes** [comment]
Hey @WadeQ 

I can see you have done the right thing.
You have changed the base branch from `master` to `develop` 👍

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -8,9 +8,6 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/AppTheme">
```
Comment (TheDancerCodes) [inline app/src/main/AndroidManifest.xml]: Hey @WadeQ 

Why did you take out the OnboardingActivity from the manifest file?

The app will not build when the code structure is like this.

Kindly do a `git pull` locally on your machine and try to run the app to see what I mean.

Comment (WadeQ) [reply app/src/main/AndroidManifest.xml]: Sorry @TheDancerCodes , i mistakenly deleted those files, how can i get them back? Or restore previous state?

**Pull request closed without merging**